src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -128,9 +128,11 @@ export const SystemCVEs = ({
         dispatch(changeSystemCVEsParameters(params));
     };
 
+    const paramsWithoutAppname = (params) => Object.fromEntries(Object.entries(params).filter((p) => p[0] !== 'appName'));
+
     useEffect(() => {
         apply(
-            isEmpty(urlParameters) ?
+            isEmpty(paramsWithoutAppname(urlParameters)) ?
```

Comment:
My guess is this function is actually changing the params to filter out the `appName=vulnerabilities` and you don't mean do to this.